cmd/create/execution_util.go

```diff
@@ -70,6 +70,29 @@ func createExecutionRequestForTask(ctx context.Context, taskName string, project
 	return createExecutionRequest(ID, inputs, authRole), nil
 }
 
+func createExecutionRequestForRelaunch(ctx context.Context, executionName string, project string, domain string, cmdCtx cmdCore.CommandContext) (*admin.ExecutionCreateRequest, error) {
```

Comment:
can we use the RelaunchExecution api here? (https://github.com/flyteorg/flyteidl/blob/master/protos/flyteidl/admin/execution.proto#L37) this way you don't have to fetch the execution and the lp

Reply:
Done